py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -47,3 +47,74 @@ def aten_ops_arange_start_step(
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
     return np.arange(*args)
+
+
+def rand_validator(rand_node: Node) -> bool:
+    dtype = rand_node.kwargs.get("dtype", None)
+    layout = rand_node.kwargs.get("layout", None)
+    if dtype is not None:
+        _LOGGER.debug(
+            f"Currently we don't support specifying output dtype, got {dtype}."
+        )
+        return False
+    if layout is not None:
+        _LOGGER.debug(f"Currently we don't support specifying layout, got {layout}.")
+        return False
+    return True
+
+
+@dynamo_tensorrt_converter(
+    torch.ops.aten.rand.default, capability_validator=rand_validator
+)
+def aten_ops_rand(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return np.random.rand(*args)
+
+
+@dynamo_tensorrt_converter(
+    torch.ops.aten.randn.default, capability_validator=rand_validator
+)
+def aten_ops_randn(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return np.random.randn(*args)
```

Comment:
This fails on my machine, since `np.random.randn` does not accept tuples, and `args` is a tuple containing a tuple:
```python
>>> args = ((1, 2, 3,),)
>>> np.random.randn(*args)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mtrand.pyx", line 1270, in numpy.random.mtrand.RandomState.randn
  File "mtrand.pyx", line 1431, in numpy.random.mtrand.RandomState.standard_normal
  File "_common.pyx", line 636, in numpy.random._common.cont
TypeError: 'tuple' object cannot be interpreted as an integer
```

Reply:
@gs-olive Thanks for pointing this out.
I think this got missed in the tests, since in the tests the input is None, and the compilation is not invoked.
I changedthe above to have `np.random.rand(*args[0])`. Also in the test I modified it to have an input (hacky way, I can as well use the harness.py with the below model)
```
def test_rand(self, name, op, shape_or_input):
        class TestModule(nn.Module):
            def __init__(self, rand_op, size):
                super().__init__()
                self.rand_op = rand_op
                self.size = size
            def forward(self, x):
                b = x + 1
                self.size[0] = b
                return self.rand_op(self.size)
```
But right now I am running into segmentation fault. I am looking into this further.